_posts/2021-01-22-sars-cov-2-variant-analysis.md

```diff
@@ -0,0 +1,54 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: 'Variant Analysis of SARS-CoV-2 Sequencing Data'
+tags: [galaxy, COVID-19, tools, workflows]
+---
+
+With the recent appearance of novel SARS-CoV-2 lineages in the UK, South Africa
+and Brazil, which are suspected to display increased infectivity and/or partial
+escape from neutralizing antibodies, the need for continuous tracking of
+viral evolution through genome sequencing has received increased public
+attention.
+
+While the UK's COVID-19 Genomics Consortium (COG-UK) has been pioneering such
+monitoring since early on in the pandemic, other countries are now eager to
+establish their own networks for genomic surveillance.
+
+In the framework of covid19.galaxyproject.org, an international team of Galaxy
+developers and researchers from different disciplines (including members of the
+Galaxy EU team and of the virology department of the University of Freiburg)
+have collaborated since February last year to establish a set of workflows for
+SARS-CoV-2 genomics analysis - including the ones that are urgently needed now.
+
+The European Galaxy server offers these as
+[__public workflows__](https://usegalaxy.eu/workflows/list_published).
+Just search the list for *covid19.galaxyproject.org* and you will find our
```

Comment:
@OlegZharkov can we somehow include the tags in the link, so that we can share a link only to workflows that are tagged in a special way?

Reply:
Do you mean something like this?
https://galaxyproject.eu/news?tag=tools

Reply:
Yes.